### PR TITLE
Add a Static Pages controller

### PR DESCRIPTION
**Pull request by caovuong**
Chapter 3

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,3 +1,6 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  def hello
```
Comment (qtsoul95) [inline app/controllers/application_controller.rb:4]: cách dòng ra

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+  	render html: "hello, world!"
```
Comment (qtsoul95) [inline app/controllers/application_controller.rb]: indent

File: app/views/static_pages/help.html.erb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<% provide :title, "Help" %>
+<!DOCTYPE html>
+<html>
+  <head>
+  	<title><%= yield :title %> | Ruby on Rails Tutorial Sample App</title>
```
Comment (qtsoul95) [inline app/views/static_pages/help.html.erb]: indent

**Comment by qtsoul95** [comment]
ok

File: app/views/static_pages/help.html.erb [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  <head>
+    <title><%= yield :title %> | Ruby on Rails Tutorial Sample App</title>
+  </head>
+  <body>
+    <h1>Help</h1>
+    <p>
+      Get help on the Ruby on Rails Tutorial at the
+      <a href="http://www.railstutorial.org/help">Rails Tutorial help
+      	section</a>.
+     	To get help on this sample app, see the
```
Comment (DuongNT1512) [inline app/views/static_pages/help.html.erb]: indent

File: app/views/static_pages/about.html.erb [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+<p>
+  The <a href="http://www.railstutorial.org/"><em>Ruby on Rails
+    Tutorial</em></a> is a
+  <a href="http://www.railstutorial.org/book">book</a> and
+  <a href="http://screencasts.railstutorial.org/">screencast series</a>
+  to teach web development with
+  <a href="http://rubyonrails.org/">Ruby on Rails</a>.
+  This is the sample application for the tutorial.
+</p>
+
```
Comment (qtsoul95) [inline app/views/static_pages/about.html.erb]: end file

File: app/views/static_pages/help.html.erb [hunk 6]
```diff
@@ -0,0 +1,10 @@
+<% provide :title, "Help" %>
+<h1>Help</h1>
+<p>
+  Get help on the Ruby on Rails Tutorial at the
+  <a href="http://www.railstutorial.org/help">Rails Tutorial help
```
Comment (qtsoul95) [inline app/views/static_pages/help.html.erb]: indent

**Pull request merged**